### PR TITLE
Release documentation and notes

### PR DESCRIPTION
**Pull request by daw3rd**
## Why are these changes needed?
Updates to release documentation
Add release notes for 0.2.1 release

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Overview 
 Release are created from the main repository branch using the version
 numbers, including an intermediate version suffix, 
-defined in `.make.versions`.
+defined in `.make.versions`
 The following points are important:
 
 1. In general, common a version number is used for all published pypi wheels and docker images.
```
Comment (touma-I) [inline RELEASE.md]: do you mean: a common (it is written common a)

File: RELEASE.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Overview 
 Release are created from the main repository branch using the version
```
Comment (touma-I) [inline RELEASE.md]: Should that read "Release is" or "Releases are"  ?

File: RELEASE.md [hunk 3]
```diff
@@ -20,29 +20,35 @@ allows intermediate publishing from the main branch using version X.Y.Z.dev\<N\>
 ## Cutting the release
 Creating the release involves
 
+1. Edit the `release-notes.md` to list major/minor changes
 1. Creating a release branch and tag and updating the main branch versions.
 1. Creating a github release from the release branch and tag.
 1. Building and publishing pypi library wheels and docker registry image.
```
Comment (touma-I) [inline RELEASE.md:26]: Is this missing the step for creating a new dev branch that increments the release number ?

Also should we use Create a ..., Build and publish... similar to first bullet.

**Review by touma-I — APPROVED** [review]
Looks good except for some gramatical comments that you can choose to ignore. :-)

**Review by touma-I — APPROVED** [review]
@daw3rd There are a couple. of comments related to grammar. We can merge with or without. Let me know. Thanks

File: RELEASE.md (hunk 1 above)
Comment (daw3rd) [reply RELEASE.md]: Done. Thanks.

File: RELEASE.md (hunk 2 above)
Comment (daw3rd) [reply RELEASE.md]: Done.  Thanks.

File: RELEASE.md (hunk 3 above)
Comment (daw3rd) [reply RELEASE.md:26]: i think that is covered in 2 `...and updating the main branch versions.`

**Pull request merged**